32_awk_lab/awklab.txt

```diff
@@ -21,6 +22,6 @@ p2p0: flags=8843 mtu 2304
 
 Write an awk one liner that splits this file into multiple file, one for each block, so after running the script we'll have a file named "en3" with the lines of the first block, and another file named "p2p0" with the lines of the second block.
 
-
+awk '{print $0 " "> "file" NR}' RS='-\\|' /sbin/ifconfig
```

Comment:
You first need to match the block - [here](https://github.com/tocodeil/unix1-course-examples/issues/73) Ynon gave some great hints. Here's another one:
``` ifconfig | awk '/^[a-z]/ {file_name=$1 }  {print file_name ":"  $0}'```